### PR TITLE
Require the Delegate to restrict scramble access to groups before they start.

### PR DESCRIPTION
**Pull request by lgarron**
@AlbertoPdRF and @Laura-O, could you review?

### CODE REVIEW

File: wca-guidelines.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ To be more informative, each Guideline is classified using one of the following
 - 1c3++) [ADDITION] Physical records (e.g. score sheets, written solutions from Fewest Moves Solving) must be retained for a minimum of one month. Digital records (e.g. scramble sequences, results) must be retained permanently.
 - 1c3b+) [CLARIFICATION] If there are multiple groups, it is not necessary to identify which competitor was in which groups.
 - 1c4+) [RECOMMENDATION] Corrections to the results should be available within one week of the competition date.
+- 1c8+) [EXAMPLE] The Delegate should only give access to scrambles for future groups to other members of the organization team if they will be temporarily unavailable (e.g. competing themselves), and should give others access to the minimum possible amount of scrambles in this case.
```
Comment (Laura-O) [inline wca-guidelines.md]: I think `scrambles` should be `scramble sequences` to have a consistent terminology.

Additionally, why `should` and not `must`?

**Review by Laura-O — COMMENTED** [review]
Two comments/questions. The rest LGTM!

**Comment by xsrvmy** [comment]
Does this imply that the delegate should email each scramble group at a time to each scrambling station separately? I have never ever seen this done at a local competition.

**Comment by Laura-O** [comment]
> Does this imply that the delegate should email each scramble group at a time to each scrambling station separately? I have never ever seen this done at a local competition.

But you have seen delegates emailing all scrambles? That's interesting, tell us more about it.

**Comment by xsrvmy** [comment]
> > Does this imply that the delegate should email each scramble group at a time to each scrambling station separately? I have never ever seen this done at a local competition.
> 
> But you have seen delegates emailing all scrambles? That's interesting, tell us more about it.

When I scramble at competitions it has always been to scrambler's responsibility to open the correct pdf. Of course the delegate/organizer will tell us to switch groups. However scramblers do need to be approved by the delegate/organizer first.

**Review by AlbertoPdRF — APPROVED** [review]
Apart from @Laura-O's comments, LGTM!

File: wca-guidelines.md (hunk 1 above)
Comment (lgarron) [reply wca-guidelines.md]: Done. Since this is in the Guidelines, I've also changed this from `[EXAMPLE]` to `[ADDITION]`.

**Comment by xsrvmy** [comment]
Shouldn't a MUST entry go in the regulations instead of the guidelines?
Also, if a delegate accidentally fails to respect this regulation before the new encrytion scheme is used in tnoodle, does the entire comp become disqualified?

**Comment by lgarron** [comment]
> Shouldn't a MUST entry go in the regulations instead of the guidelines?

It should, but sometimes we make exceptions. This one's an addition to a SHOULD in the Regulations, and I don't feel comfortable codifying the exception until we've seen what happens in practice.

> Also, if a delegate accidentally fails to respect this regulation before the new encrytion scheme is used in tnoodle, does the entire comp become disqualified?

No (unless scrambles were leaked).

**Pull request merged**